models/Post.php

```diff
@@ -104,6 +104,7 @@ class Post extends Model
     public $preview = null;
 
     /**
+     * Set value for the field user to current user in the create context
```

Comment:
@rubenvanerk Could you move this comment line and the one below inside the function just above where those changes take place, and change this docblock's comment to say `Update fields based on current user.`?